scripts/genRef.py

```diff
@@ -139,6 +139,9 @@ def seeAlsoList(apiName, explicitRefs=None, apiAliases=[]):
                 if dependency is not None:
                     refs.add(dependency)
 
+    # Filter references based on API conventions
+    refs = set([ref for ref in refs if not conventions.filter_ref(ref)])
+
```

Comment:
I think you could also, slightly more pythonically, do

> refs = set(filter(lambda ref: conventions.filter_ref(ref), refs))

(which I just learned about on asking "hm, seems like there should be a builtin for this" :-).